src/Shell/UtilityManager.php

```diff
@@ -12,8 +12,8 @@
  */
 class UtilityManager
 {
-    const UTILITY_TIMEOUT = 'timeout';
-    const UTILITY_BASH = 'bash';
+    public const UTILITY_TIMEOUT = 'timeout';
+    public const UTILITY_SHELL = 'sh';
```

Comment:
I like this change and agree it's the correct course of action here, but is this not a backwards incompatible change? What version are we planning to release this in?

Reply:
Good catch - added the previous constant as deprecated